app/ports/models/stats.py

```diff
@@ -38,11 +40,16 @@ def populate(cls, json_object, timestamp):
         uuid_obj, created = UUID.objects.get_or_create(uuid=json_object['id'])
         sub = Submission()
         sub.user = uuid_obj
-        sub.os_version = json_object['os'].get('osx_version')
+        os_version = json_object['os'].get('osx_version')
+        cxx_stdlib = json_object['os'].get('cxx_stdlib')
+        if cxx_stdlib is None and compare_versions(os_version, '10.9'):
```

Comment:
`compare_versions` as a name still doesn't tell if one is greater than the other.

Reply:
Thanks, do you have some suggestion?

Reply:
I mean, now that we got a `LooseVersion` implemented,  you can really just do a `if ... and LooseVersion(v1) >= LooseVersion(v2)` check. 
But what was the reason behind choosing `LooseVersion` and `StrictVersion` in our case? I wasn't actually able to decide, so I don't have an answer either.

Reply:
`LooseVersion` gives just some more flexibility in choosing versions as compared to `StrictVersion`. I thought that sometimes port versions don't follow a good nomenclature so using `LooseVersion` might be better.

Reply:
except that we are not using it for comparing port versions. But okay, I got the idea.

Reply:
I have removed the function `compare_versions` until we need it somewhere else.